# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gtkextra2/fix_configure_in.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./configure.in.old	Mon Jul 22 19:09:14 2002
+++ ./configure.in	Sun Feb  2 22:46:21 2003
@@ -4,7 +4,7 @@
 # Save this value here, since automake will set cflags later
 cflags_set=${CFLAGS+set}
 
-AC_DIVERT_PUSH(AC_DIVERSION_NOTICE)dnl
+AC_DIVERT_PUSH(NOTICE)dnl
 # Making releases:
 #   GTK_EXTRA_MICRO_VERSION += 1;
 #   GTK_EXTRA_INTERFACE_AGE += 1;
@@ -20,7 +20,7 @@
 GTK_EXTRA_BINARY_AGE=0
 GTK_EXTRA_VERSION=$GTK_EXTRA_MAJOR_VERSION.$GTK_EXTRA_MINOR_VERSION.$GTK_EXTRA_MICRO_VERSION
 dnl
-AC_DIVERT_POP()dnl
+AC_DIVERT_POP()dnl
 
 AC_SUBST(GTK_EXTRA_MAJOR_VERSION)
 AC_SUBST(GTK_EXTRA_MINOR_VERSION)
